src/strings-overrides.js

```diff
@@ -21,41 +21,16 @@ addFilter(
 		);
 
 		if (
-			isUnsupportedBlockEditorSupported === false &&
+			isUnsupportedBlockEditorSupported ||
 			canEnableUnsupportedBlockEditor
```

Comment:
The condition to check here is that either UBE is available (`isUnsupportedBlockEditorSupported` = true) or if unavailable, it can be enabled (`canEnableUnsupportedBlockEditor`= true`).